packages/node_modules/samples/browser-read-status/index.html

```diff
@@ -116,17 +116,17 @@ <h2>Monitor who has read messages in a user's Webex Teams spaces</h2>
                   </fieldset>
                 </form>
               </div>
-              
-            </center>  
-            
-            <a href="./explanation.html" target="_blank">What's happening in this demo?</a> 
+
```

Comment:
Autoformat is awesome.

Reply:
I dont know what's autoformatting it lol